charts/kubeflow-trainer/values.yaml

```diff
@@ -0,0 +1,125 @@
+#
+# Copyright 2024 The Kubeflow authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+# Default values for trainer.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+# -- String to partially override release name.
+nameOverride: ""
+
+# -- String to fully override release name.
+fullnameOverride: ""
+
+jobset:
+  # -- Whether to install jobset as a dependency managed by trainer.
+  # This must be set to `false` if jobset controller/webhook has already been installed into the cluster.
+  install: true
+
+# -- Common labels to add to the resources.
+commonLabels: {}
+
+image:
+  # -- Image registry.
+  registry: docker.io
+  # -- Image repository.
+  repository: kubeflow/trainer-controller-manager
+  # -- Image tag.
+  # @default -- If not set, the chart appVersion will be used.
+  tag: latest
+  # -- Image pull policy.
+  pullPolicy: IfNotPresent
+  # -- Image pull secrets for private image registry.
+  pullSecrets: []
+  # - name: <secret-name>
+
+controller:
+  # -- Number of replicas of controller.
+  replicas: 1
```

Comment:
@kubeflow/wg-training-leads @Electronic-Waste @ChenYi015 Do we support leader election in TrainJob controller?